packages/admin/cms-admin/src/contentScope/ContentScopeSelect.tsx

```diff
@@ -107,7 +107,14 @@ export function ContentScopeSelect({
                         <Domain />
                     </ListItemIcon>
                     <ListItemText
-                        primaryTypographyProps={{ variant: "body2", fontWeight: "inherit" }}
+                        slotProps={{
+                            primary: {
+                                variant: "body2",
```

Comment:
as shown in the Figma design and screenshots, simply switching the variant should work and be enough.

Selected variant should be "subtitle2"

![Screenshot 2025-06-30 at 14 37 28](https://github.com/user-attachments/assets/dd672897-e2e0-467f-8b2b-c3c51e7e941b)

and normal version should be body2:
![Screenshot 2025-06-30 at 14 37 32](https://github.com/user-attachments/assets/58159b89-8e7c-40bb-ad4b-bf454af26a36)


In my opinion following [change](https://github.com/vivid-planet/comet/pull/3736/files#diff-cafd62486f644eb13187c898127d64882a128833333d1b45646a5271152be275R13) on the MuiListItemText styling introduced this bug - remove the theme styling of the primary should fix this.

Reply:
Good point, but as you pointed out, setting the variant probably won't apply the correct `fontWeight` without changing how `MuiListItemText` is styled/themed. But changing this is out-of-scope imo, so I'd go with the current implementation.